app/uk/gov/hmrc/incorporatedentityidentificationfrontend/views/capture_company_number_page.scala.html

```diff
@@ -26,36 +28,39 @@
         govukButton: GovukButton,
         formWithCSRF: FormWithCSRF,
         inputText: inputText,
+        errorSummary: errorSummary
 )
 
 @(formAction: Call, form: Form[String])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @layout(title = title(messages("capture-company-number.title"), form)) {
-    @govukFieldset(Fieldset(
-        legend = Some(Legend(
-            content = Text(Messages("capture-company-number.title")),
-            classes = "govuk-fieldset__legend--xl",
-            isPageHeading = true
-        )),
-        html = html
-    ))
+    @errorSummary(form.errors)
 
-}
+        @govukFieldset(Fieldset(
```

Comment:
I don't think there needs to be a fieldset here because they're used when there is more than one question on the page. Check out the STI capture nino page - will be very similar to that.